aws-observability-terraform/app-modules/sns/app.tf

```diff
@@ -0,0 +1,188 @@
+module "sns_module" {
+  source = "SumoLogic/sumo-logic-integrations/sumologic//sumologic"
+
+  access_id   = var.access_id
+  access_key  = var.access_key
+  environment = var.environment
+
+  # ********************** No Metric Rules for API Gateway ********************** #
+
+  # ********************** Fields ********************** #
+  /***
+   managed_fields = {
+     "TopicName" = {
+       field_name = "topicname"
+       data_type  = "String"
+       state      = true
+     }
+   }
+***/
+  # ********************** FERs ********************** #
+  managed_field_extraction_rules = {
+    "SnsFieldExtractionRule" = {
+      name             = "AwsObservabilitySNSLogsFER"
```

Comment:
I will recommend FER name to be AwsObservabilitySNSCloudTrailLogsFER. Accordingly update resource name.